README.md

```diff
@@ -28,47 +28,49 @@ Mount or umount the root filesystem from the given VM. For example `systemd star
-| `xen_vman_default_spiceport`             | integer               | `3100`                                                              | Specify the port to listen on by the SPICE server if SPICE is enabled [¹](#xen_doc)                                                   |
-| `xen_vman_default_mouse`                 | boolean               | True                                                                | Whether SPICE agent is used for client mouse mode [¹](#xen_doc)                                                                       |
-| `xen_vman_default_share_clipboard`       | boolean               | True                                                                | Enables SPICE clipboard sharing [¹](#xen_doc)                                                                                         |
-| `xen_vman_default_spicevdagent`          | boolean               | True                                                                | Enables SPICE vdagent, this is automatically enabled when clipboard sharing is enabled [¹](#xen_doc)                                  |
-| `xen_vman_default_max_usb_redirections`  | integer (0-4)         | `4`                                                                 | Enable SPICE USB redirections [¹](#xen_doc)                                                                                           |
-| `xen_vman_default_additonal_xen_options` | xl.cfg key value dict | `[]`                                                                | Add additional Xen options                                                                                                            |
-| `xen_vman_default_auto_install`          | boolean               | `True`                                                              | Install specified OS automatically, when no xen configuration is found                                                                |
+| Name                                     | Type                  |          : Required :         | Default                                                             | Description                                                                                                                           |
+|------------------------------------------|-----------------------|:-----------------------------:|---------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| `xen_vman_xl_path`                       | string                |               N               | _auto determined_                                                   | Fully qualified path of the xl command                                                                                                |
```

Comment:
Please consider using one column for Required/Default like in the roles written by @dasJ, @Scriptkiddi  and me. I can't force you, but I want to appeal to you and would really like to see it.
I think it's a cleaner way of structuring the variable table, it saves space, gives a better overview and doesn't use too many columns. Besides, in almost all cases, a variable having a default value and a variable not being required occur together, so having two columns for this information is redundant and can be confusing.